Tools/autotest/quadplane.py

```diff
@@ -446,6 +446,7 @@ def fly_qautotune(self):
 
     def takeoff(self, height, mode):
         """climb to specified height and set throttle to 1500"""
+        self.set_current_waypoint(0)
```

Comment:
Hmmm.  I think this is too broad a change - too much magic stuff happening in something called "takeoff".  I can live with it for now, however.

Reply:
I think the underlying problem is the chaining of tests without a state reset in autotest. The tests run fine alone, but when chained we can be in landing seq when arming due to another test